how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,296 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core import Workspace\n",
+    "from azureml.pipeline.core.schedule import ScheduleRecurrence, Schedule\n",
+    "from azureml.pipeline.core import PublishedPipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Schedule Operations\n",
+    "### Get published pipeline ID\n",
+    "- [ws](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.workspace.workspace?view=azure-ml-py): The workspace object this Schedule will belong to. It need to have at least one published pipeline. [(How to publish pipeline)](https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-how-to-use-pipeline-drafts.ipynb) \n",
+    "- [pub_pipeline](https://docs.microsoft.com/en-us/python/api/azureml-pipeline-core/azureml.pipeline.core.pipeline.pipeline?view=azure-ml-py): A published pipeline which will be scheduled.\n",
+    "- config_path: The path of the workspace's config file, witch can be downloaded on portal."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config_path = \"config.json\"\n",
+    "ws = Workspace.from_config(path=config_path)\n",
+    "all_pub_pipelines = PublishedPipeline.list(ws)\n",
+    "print(\"Published pipelines found in the workspace:\")\n",
+    "for pub_pipeline in all_pub_pipelines:\n",
+    "    print(pub_pipeline.id)\n",
+    "pub_pipeline_id = all_pub_pipelines[0].id\n",
+    "\n",
+    "print(\"Published pipeline id to be used for Schedule operations: {}\".format(pub_pipeline_id))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a schedule with pipeline parameter built-in functions\n",
+    "This schedule will run on a specified recurrence interval and with pipeline parameter built-in functions. The functions will be resolved dynamically along with run triggered.\n",
+    "\n",
+    "See below table for supported built-in functions in schedule, the functions can be invoked with an identifier `@` at the beginning of the function string, otherwise treated as raw string or raise an error. For more information, please read *Deal with invalid functions* At the bottom of the doc.\n",
+    "\n",
+    "Note: `?` means optional.\n",
+    "\n",
+    "| Function    | Description |\n",
+    "| ----------- | ----------- |\n",
+    "| `trigger().scheduledTime` | Return time at which the trigger was scheduled to invoke the pipeline run. Consistent with the time you specific in `ScheduleRecurrence` instance.\n",
+    "| `trigger().startTime` | Return time at which the trigger actually fired to invoke the pipeline run. This may be slightly later than the trigger's scheduled time.\n",
+    "| `addDays('<timestamp>', <days>, '<format>'?)`    | Add a number of days to a timestamp.\n",
```

Comment:
Add in Description that the two trigger expressions can only be resolved in schedule scenario.